assets/benchmarks/shaders/Benchmark_SolidColor.hlsl

```diff
@@ -22,18 +25,20 @@ struct ColorParams
 #endif
 ConstantBuffer<ColorParams> Color : register(b0);
 
-struct VSOutput {
-    float4 position : SV_POSITION;
-};
-
-VSOutput vsmain(float4 Position : POSITION)
+// returns zigzag value between (0.5 ~ 1.0) in steps of 0.1 between consecutive seeds
+// this is chosen to try and reduce flashing effect when the slider is dragged slowly
+// but so that color difference is still visible to the eye between consecutive quads
+float intensity(uint32_t seed)
```

Comment:
can this be done on the cpu side instead of in the shader?

Reply:
Sure, moved this part out from shader